cmd/helm/installer/install_test.go

```diff
@@ -135,10 +135,10 @@ func TestDeploymentManifest_WithTLS(t *testing.T) {
 			t.Fatalf("%s: error %q", tt.name, err)
 		}
 		// verify environment variable in deployment reflect the use of tls being enabled.
-		if got := d.Spec.Template.Spec.Containers[0].Env[1].Value; got != tt.verify {
+		if got := d.Spec.Template.Spec.Containers[0].Env[2].Value; got != tt.verify {
```

Comment:
That's rather brittle. Did you catch it by way of the tests failing at first?
Perhaps we should scan the environment table to find the key we're expecting, and then check the corresponding value.